driver/solo/fv_phys.F90

```diff
@@ -1582,27 +1584,25 @@ subroutine fv_phys_init(is, ie, js, je, km, nwat, ts, pt, time, axes, lat)
     master = is_master()
 
 !   ----- read and write namelist -----
-    if ( file_exist('input.nml')) then
-         unit = open_namelist_file ('input.nml')
-         read  (unit, nml=sim_phys_nml, iostat=io, end=10)
+    if ( file_exists('input.nml')) then
```

Comment:
Do we need this if-test since we'll be reading from the character variable?  That should've been checked during fms_init (mpp_init) at program startup.

Reply:
`read_input_nml` checks if input.nml exists and `read_input_nml` is called within `mpp_init`, so it looks like you are right.  @uramirez8707 let me know if you see any reason we should keep this.